res/battle/graphic/pl_batt_obj/pl_batt_obj.ignore

```diff
@@ -0,0 +1,228 @@
+healthbox_doubles_opponent_2.NCER
```

Comment:
**suggestion**: Maybe we want to discuss with red about adding `lz` encoding as a switch on `nitrogfx` outputs so that we can avoid having huge `ignore` files like these.

Reply:
yeah, this would probably be helpful. i suspect there will be other large NARCs in the project with lots of compressed files

Reply:
I'll write up a PR to implement this over the course of the next couple of days.